python-checks/src/test/resources/checks/datetime_constructor_use_timezone_aware.py

```diff
@@ -36,8 +36,5 @@ def aliased_utcnow():
     # FN because we lose the FQN in the assignment
     reassigned()
 
-    from datetime.datetime import utcnow as aliased
-    aliased() # Noncompliant
-
-    from datetime.datetime import utcnow
-    utcnow() # Noncompliant
+    from datetime import datetime as aliased
+    aliased.utcnow() # Noncompliant
```

Comment:
This triggered an edge case when replacing members of `ModuleType` because we'd end up replacing the `datetime.datetime` `ClassType` with an empty `ModuleType` (after unsuccessfully trying to resolve it as a module).